src/main/kotlin/no/nav/syfo/huskelapp/kafka/HuskelappConsumer.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.syfo.huskelapp.kafka
+
+import no.nav.syfo.huskelapp.HuskelappService
+import no.nav.syfo.kafka.KafkaConsumerService
+import org.apache.kafka.clients.consumer.ConsumerRecords
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.slf4j.LoggerFactory
+import java.time.Duration
+
+class HuskelappConsumer(
+    private val huskelappService: HuskelappService
+) : KafkaConsumerService<KafkaHuskelapp> {
+
+    override val pollDurationInMillis: Long = 1000
+
+    override fun pollAndProcessRecords(kafkaConsumer: KafkaConsumer<String, KafkaHuskelapp>) {
+        val records = kafkaConsumer.poll(Duration.ofMillis(pollDurationInMillis))
+        if (records.count() > 0) {
+            processRecords(records)
+            kafkaConsumer.commitSync()
+        }
+    }
+
+    private fun processRecords(
+        consumerRecords: ConsumerRecords<String, KafkaHuskelapp>,
+    ) {
+        val (tombstones, validRecords) = consumerRecords.partition { it.value() == null }
+
+        if (tombstones.isNotEmpty()) {
+            val numberOfTombstones = tombstones.size
+            log.error("Value of $numberOfTombstones ConsumerRecord are null, most probably due to a tombstone. Contact the owner of the topic if an error is suspected")
+            COUNT_KAFKA_CONSUMER_HUSKELAPP_TOMBSTONE.increment(numberOfTombstones.toDouble())
+        }
```

Comment:
Vi sender vel ikke tombstones på denne topicen, så vet ikke om vi trenger dette?

Reply:
Nei, jeg tenker vi kan fjerne det.

Reply:
Endret nå, prøvde å ta i bruk en `requireNoNulls`, den kaster feil hvis det plutselig skulle komme null-verdier her.
_Returns an original collection containing all the non-null elements, throwing an IllegalArgumentException if there are any null elements._

Reply:
Nice!